dev/com.ibm.ws.com.unboundid/src/com/ibm/ws/com/unboundid/InMemoryLDAPServer.java

```diff
@@ -292,7 +294,7 @@ public int importFromLDIF(boolean clear, String path) throws LDAPException {
      * @param listenerName - The name of the listener for which to retrieve the listen port. It may be null in order to obtain the listen port for the first active listener
      * @return The configured listen port for the specified listener, or -1 if there is no such listener or the listener is not active.
      */
-    public int getListenPort(String listenerName) {
-        return ds.getListenPort(listenerName);
+    public int getSecureListenPort() {
```

Comment:
Not a fan of this or the `getListenPort()` method names. I would rather they were `getLdapPort()` and `getLdapsPort()`.

Not sure if you can rename `getListenPort()` without making changes in WS-CD-Open. If WS-CD-Open calls `getListenPort()`, deprecate that method and create these two others and then replace any calls in Open Liberty. If WS-CD-Open doesn't call this method, then you can remove the method entirely.